### PR TITLE
Adding support for DALEX package

### PR DESCRIPTION
**Pull request by adrianstando**
Issue #60

### CODE REVIEW

File: R/shapviz_methods.R
Comment (mayer79) [inline R/shapviz_methods.R:1]: We actually do not need to care about the "collapse" logic. It modifies the input X on initialization. Afterward, the shapviz objects forget about it. So also no need for a `get_collapse()` function.

**Review by mayer79 — CHANGES_REQUESTED** [review]
Not fully through yet, but this is a good start :-)

File: R/shapviz_methods.R
Comment (adrianstando) [reply R/shapviz_methods.R:1]: Okay, I see it. I believed it is stored as an attribute

File: R/shapviz.R [hunk 1]
```diff
@@ -240,6 +241,41 @@ shapviz.treeshap <- function(object, X = object[["observations"]],
...
+    X <- as.data.frame(aggregated[, c('variable_value')])
+    X <- as.data.frame(t(X))
+    colnames(X) <- rownames(aggregated)
+    rownames(X) <- NULL
+
+    shapviz::shapviz(shaps, X = X, baseline = baseline, collapse = collapse)
+  } else {
+    stop("Incorrect object! It is neither 'shap' nor 'break_down'!")
+  }
+}
```
Comment (mayer79) [inline R/shapviz.R:277]:
```suggestion
shapviz.predict_parts <- function(object, ...) {
  if (!inherits(object, c("shap", "shap_aggregated", "break_down"))) {
    stop("Incorrect object! It is neither 'shap', 'shap_aggregated' nor 'break_down'!")
  }
  if (inherits(object, "shap")) {
    agg <- as.data.frame(object[object$B == 0, ])
    baseline <- attr(object, "intercept")
  } else {
    if (inherits(object, "shap_aggregated")) {
      agg <- object$aggregated
    } else {  # break_down
      agg <- as.data.frame(object)  # to drop additional classes
    }
    baseline <- agg[1L, "contribution"]
    agg <- agg[2:(nrow(agg) - 1L), ]     # Drop intercept and prediction
  }

  ## Problem: agg might contain all feature values as strings
  ## -> excludes sv_dependence() and beeswarms
  X <- data.frame(t(agg[["variable_value"]]))
  S <- t(agg[["contribution"]])
  colnames(X) <- colnames(S) <- agg[["variable_name"]]

  shapviz(shaps, X = X, baseline = baseline, ...)
}
```

Comment (mayer79) [reply R/shapviz.R:277]: My try to get used to predict_parts() ;). Will need a short review from your side. Rest looks fine!

**Review by mayer79 — APPROVED** [review]
One function to finish, otherwise looks good to me, thanks!

File: R/shapviz.R (hunk 1 above)
Comment (adrianstando) [reply R/shapviz.R:277]: I think this version of the function looks better :) 
Only **shaps** object in the last line has to be changed to **S**, but I will fix this with next commit.

**Comment by mayer79** [comment]
LGTM

**Pull request merged**